neo4j/bookmarks.go

```diff
@@ -12,15 +23,35 @@ Let s1, s2, s3 be Session interfaces. You can easily causally chain the sessions
 The server will then make sure to execute all transactions in s4 after any that were already executed in s1, s2, or s3
 at the time of calling LastBookmarks.
 */
-func CombineBookmarks(slices... []string) []string  {
+func CombineBookmarks(bookmarks... Bookmarks) Bookmarks  {
 	var lenSum int
-	for _, s := range slices {
-		lenSum += len(s)
+	for _, b := range bookmarks {
+		lenSum += len(b)
 	}
 	res := make([]string, lenSum)
 	var i int
-	for _, s := range slices {
-		i += copy(res[i:], s)
+	for _, b := range bookmarks {
+		i += copy(res[i:], b)
 	}
 	return res
 }
+
+/*
+BookmarksToRawValues exposes the raw server-side bookmarks.
+
+You should not need to use this method unless you want to serialize bookmarks.
+See Session.LastBookmarks and CombineBookmarks for alternatives.
+ */
+func BookmarksToRawValues(bookmarks Bookmarks) []string {
+	return bookmarks
+}
+
+/*
+BookmarksFromRawValues creates Bookmarks from raw server-side bookmarks.
+
+You should not need to use this method unless you want to de-serialize bookmarks.
+See Session.LastBookmarks and CombineBookmarks for alternatives.
+*/
+func BookmarksFromRawValues(values... string) Bookmarks {
```

Comment:
nit: `ParseBookmarks` maybe?

Reply:
Maybe. I was thinking two things: 
 * it should somewhat be obvious that this is the inverse operation to `BookmarksToRawValues` (or whatever we decide to call it)
 * if it starts with "Bookmarks" it will be easier to discover through your IDE's auto-completion.